### ISSUE TITLE
[ROOTS] Accessibility/behaviour of buttons inconsistent accross site

### ISSUE THREAD

**Issue by arc64** [liqd/a4-roots#9]
**Behavour:**
Buttons have inconsistent hover, active, focus behaviour.

The best place to see button behaviour is in the [component library](/components/#cl-buttons). [Defakts example](https://defakts-dev.liqd.net/components/#cl-buttons).

**Expected Behaviour:** 
Make all buttons, including the donate box button consistent and accessible. Including hover, active and focus states. 

**Tasks:**

- [ ] Design a focus, active and hover state for buttons that can be implemented consistently.
- [ ] Implement new styles for buttons.

Note: this is an issue in Adhocracy+, Defakts too. I'm not sure if it is an issue in other forks.

**Comment by arc64** [comment]
@CarolingerSeilchenspringer I noticed that ROOTs, Defakts and Adhocracy buttons are styled inconsistently and are not accessible as they don't have consistent hover, active and focus states. I searched but didn't find an issue raised for this already, so I've raised it here against ROOTS.

**Comment by CarolingerSeilchenspringer** [comment]
@arc64 I will hand over this issue to @t-j-q so that things can be changed in A+ and then rebased in ROOTS and defakts. Not sure about priority though.

**Comment by t-j-q** [comment]
Thank you @arc64! 

It's definitely in the backlog. Although this appears like a bug, I would rather classify it as an enhancement here on GitHub. We agreed on using the tag enhancement for things that haven't changed because of recent developments like stories or releases. As this is a more general finding, that- in all probability- isn't caused by such events, enhancement is more suitable here. @CarolingerSeilchenspringer has to decide whether we want to keep the issue for the defakts repo, but as correctly stated, this needs to be fixed in a+ in the first place due to the rebase logic.

**Comment by CarolingerSeilchenspringer** [comment]
@t-j-q @arc64 if already on a+ list I would then close it here for ROOTS.
